### DISCUSSION TITLE
balancing with technic not working with sample balancer program

### DISCUSSION THREAD

**Discussion by injeniero** [pybricks/support#596] [Q&A]
Hi!

I was trying the sample at https://github.com/pybricks/pybricks-projects/blob/master/sets/mindstorms-robot-inventor/other-models/balancer/main.py with the technic hub and it doesn't work as shown on the video, it simply falls and start running away. I started printing the values and they seem to be quite off, I was expecting values between [-100, 100] for final duty, but they are like -2509.564. I'm using the Motors from lego Boost, I would think some constants would need to be adjusted for this new configuration, but I would appreciate some guidance on it.

Thanks
![IMG_2431](https://user-images.githubusercontent.com/739218/149549961-61fb9958-784e-49f4-b8ba-8bd6eacf2e0b.jpg)

**Comment by SpudGunMan** [comment]
off hand, the hub/sensor and motors are different so there will be some adjustments needed I assume I don't have this hub to test with. check out the PID balancing robot scripts and see if you can adapt them (having the PID values handy might assist in the balancing act)
